packages/skygear-core/lib/cloud/asset.js

```diff
@@ -228,3 +229,14 @@ export function getSigner() {
   }
   return sharedSigner;
 }
+
+export class Asset extends BaseAsset {
+  constructor(attrs) {
+    super(attrs);
+
+    let base64 = attrs.base64;
+    if (base64) {
+      this.file = Buffer.from(base64, 'base64');
```

Comment:
Is it possible to move this into `/lib/asset.js` (the non-cloud part of the asset)? I hope it is possible to detect if `Buffer` is available and automatically use `Buffer` if it is available.